org.civicrm.reqangular/src/common/services/api/contract.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'common/modules/apis',
+  'common/services/api'
+], function (_, apis) {
+  'use strict';
+
+  apis.factory('api.contract', ['$log', 'api', function ($log, api) {
+    $log.debug('api.contract');
+
+    /**
+     * Contracts data will have key 'api.HRJobContract.getfulldetails'
+     * which is normalized with a friendlier 'details' key
+     *
+     * @param  {Object} contract
+     * @return {Object}
+     */
+    function storeDetails (contract) {
+      var clone = _.clone(contract);
+      var info = clone['api.HRJobContract.getfulldetails'];
+
+      clone.info = info;
```

Comment:
Reduced code. However, not sure I can call it "details" because the API populates and object containing another prop "details". Please see the structure of the obj:

![image](https://user-images.githubusercontent.com/3973243/27283404-ff0842dc-54eb-11e7-9b61-a8ad88e22948.png)